AutomatedTesting/Gem/PythonTests/assetpipeline/metadata_relocation_tests/TestSuite_Main.py

```diff
@@ -0,0 +1,69 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+import shutil
+import zipfile
+
+import pytest
+from ly_test_tools.environment import file_system
+from ly_test_tools.o3de.editor_test import EditorTestSuite, EditorSingleTest
+
+
+def cleanup_test_files(workspace, test_file_names):
+    for test_file_name in test_file_names:
+        file_system.delete([os.path.join(workspace.paths.engine_root(), "AutomatedTesting", test_file_name)],
+                           True, True)
+
+
+def to_meta_file(file):
+    return file + ".meta"
+
+
+def to_material_file(file):
+    return file + ".material"
+
+
+def original_file_set(file):
+    set = [to_material_file(file), to_meta_file(to_material_file(file))]
+    return set
+
+
+def renamed_file_set(file):
+    file = file + "_renamed"
+    set = original_file_set(file)
+    return set
+
+
+@pytest.mark.SUITE_main  # Marks the test suite to be run as MAIN
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])  # This test works on Windows editor
+@pytest.mark.parametrize("project", ["AutomatedTesting"])  # Use AutomatedTesting project
+class TestAutomation(EditorTestSuite):

```

Comment:
I think it will be useful to have some asset renaming tests in main, but I'm wondering if we want any that load the Editor like this to be in Periodic?

Maybe we should start with the tests in the Periodic suite and move things to Main based on performance and impact?

Reply:
Moved